neofetch

```diff
@@ -1631,14 +1631,15 @@ get_wm() {
     else
         case "$os" in
             "Mac OS X")
-                ps_line="$(ps -e | grep -o '[S]pectacle\|[A]methyst\|[k]wm\|[c]hun[k]wm\|[y]abai')"
+                ps_line="$(ps -e | grep -o '[S]pectacle\|[A]methyst\|[k]wm\|[c]hun[k]wm\|[y]abai\|[R]ectangle')"
```

Comment:
```suggestion
                wm_names=(
                    "[S]pectacle"
                    "[A]methyst"
                    "[k]wm"
                    "[c]chun[k]wm"
                    "[y]abai"
                    "[R]ectangle"
                )

                regex_part=$( IFS='|'; echo "${wm_names[*]}" )

                ps_line=$(ps -e | grep -Eio "$regex_part")
```

That _might_ work better and make it easier to add further wms.

Reply:
I'm not sure what the purpose of the `[x]` pattern was since it's only a single letter in each case, but I left that alone in case there is some regex magic I am unaware of

Reply:
I was wondering about that too, but I'm pretty sure it's so that `ps-e | grep …` doesn't match itself:
<img width="605" alt="Screen Shot 2019-08-06 at 11 17 36 PM" src="https://user-images.githubusercontent.com/1753533/62592239-6e850400-b8a0-11e9-9ee2-567bd081b331.png">

Reply:
Oh! That rings a bell!  Nice catch. I also inadvertently made it case-insensitive with my unnecessary `-i` flag.